PlantPal/PlantPal/WelcomeScreen/WelcomeScreenView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  ContentView.swift
+//  PlantPal
+//
+//  Created by Marcin Dytko on 07/07/2024.
+//
+
+import SwiftUI
+// MARK: - VIEW
+
+struct WelcomeScreenView: View {
+    // MARK: - Properties
+    
+    @ObservedObject var viewModel: WelcomeScreenViewModel
+    
+    // MARK: - BODY
+    var body: some View {
+        NavigationStack {
+            MainStack {
+                VStack {
+                    Image(.plant)
+                    
+                    Text("PlantPal AI")
+                        .font(.custom("Merriweather-Regular", size: 32))
+                        .padding(.bottom, 8)
+                    
+                    Text("Your pocket botanist")
+                        .font(.custom("Merriweather-Regular", size: 18))
+                        .padding(.bottom, 4)
+                    
+                    Divider()
+                        .background(.divider)
+                        .frame(width: 336)
+                        .padding(.bottom, 8)
```

Comment:
Lepiej gdybyśmy mieli tutaj taki komponent:
```suggestion
                    PalDivider()
```
Nazwa dowolna. I w środku tego widoku, żeby on był oparty o Rect, a nie ten natywny Divider. I nie ustawiajmy też na sztywno szerokości. Niech ona sama się ustawia zawsze na pełną szerokość.